bindgen/ir/types/ArrayType.h

```diff
@@ -2,17 +2,20 @@
 #define SCALA_NATIVE_BINDGEN_ARRAYTYPE_H
 
 #include "Type.h"
+#include <stdint-gcc.h>
```

Comment:
This `#include` breaks compilation on macOS. Doesn't look like it is needed. Is it an IntelliJ auto-import?

Reply:
Yes, it can be removed

Reply:
Done in #112